launcher/ui/pages/global/APIPage.cpp

```diff
@@ -137,8 +145,8 @@ void APIPage::loadSettings()
     ui->msaClientID->setText(msaClientID);
     QString metaURL = s->get("MetaURLOverride").toString();
     ui->metaURL->setText(metaURL);
-    QString curseKey = s->get("CFKeyOverride").toString();
-    ui->curseKey->setText(curseKey);
+    QString flameKey = s->get("FlameKeyOverride").toString();
```

Comment:
Wouldn't changing this remove the current override for people who already have overrides?

Reply:
Migration code: https://github.com/PolyMC/PolyMC/blob/fb10a90fe3f1c730af7e2cd7905a3f154407ab85/launcher/Application.cpp#L711-L721

Reply:
oh nvm